assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -414,10 +438,21 @@ def save_as_mlflow(self):
             package_with_version = _get_pinned_requirement(package_name)
             self._extra_pip_requirements.append(package_with_version)
 
-        return super()._save(
-            code_paths=[VisionMLflowConvertor.PREDICT_FILE_PATH, VisionMLflowConvertor.VISION_UTILS_FILE_PATH],
-            segregate=True,
-        )
+        if self._model_flavor == "OSS":
+            vision_model = transformers.pipeline(task=self._task, model=str(self._model_dir))
+            vision_model.model.config = config
+            image_path = str(Path(__file__).parent.parent / "common" / "test_image.jpg")
+            self._signatures = infer_signature(
```

Comment:
possible to infer without adding a test img?

Reply:
We could do it like this:
    buffered = BytesIO()
    im = torch.rand((256, 256, 3))
    image = Image.fromarray(im.numpy().astype('uint8'), 'RGB')
    image.save(buffered, format="JPEG")
    img_str = base64.b64encode(buffered.getvalue()).decode("utf-8")
    return img_str
    
    But have to make sure bytestio and base64 packages are there. Would it be better to add these packages and get a mock_image instead??

Reply:
Fixed this, can you please take a look?
ref run: https://ml.azure.com/experiments/id/69a916d8-6a8d-42cc-9145-ec05e2228892/runs/f922069b-ba70-4b7a-9aec-dd0e5f42e254?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/madhumaddi-rg/providers/Microsoft.MachineLearningServices/workspaces/madhumaddi-hd-onboard&tid=72f988bf-86f1-41af-91ab-2d7cd011db47